lib/statics/kompasText.dart

```diff
@@ -15,9 +15,11 @@ class KompasText extends StatelessWidget {
   Widget build(BuildContext context) {
     return Text(
       text,
-      softWrap: true,
       style: style,
-      textAlign: textAlign
+      textAlign: textAlign,
+      overflow: TextOverflow.ellipsis,
+      maxLines: 10,
```

Comment:
After indroducing `overflow`, `maxLines` had to be specified as well because all long texts would end in `...` instead of wrap to the next line (`overflow` property is called before `softWrap` in `Text`)

10 was a number that prevented the longest text in infoView on Nokia1 from being cut with `...`

However, when I think of it now- if we want to go all the way through with `kompasText` widget and apply it to most of the texts in the app, I guess we would need some other solution.

I propose to add this to the description in #64

Reply:
Ok. Sounds good.

Reply:
Unfortunately now we get the error `RenderFlex overflowed by ... pixels on the right` when resizing screen width on the info page when "Osiągnięcia" is expanded.